### PR TITLE
added football standings and phone specifications API

### PR DESCRIPTION
**Pull request by azharimm**
update order football standing API

- [v] My submission is formatted according to the guidelines in the [contributing guide](/CONTRIBUTING.md)
- [v] My addition is ordered alphabetically
- [v] My submission has a useful description
- [v] The description does not end with punctuation
- [v] Each table column is padded with one space on either side
- [v] I have searched the repository for any relevant issues or pull requests
- [v] Any category I am creating has the minimum requirement of 3 items
- [v] All changes have been [squashed][squash-link] into a single commit

[squash-link]: <https://github.com/todotxt/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit>

### CODE REVIEW

**Review by Giahuynh1402 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1048,6 +1049,7 @@ API | Description | Auth | HTTPS | CORS |
 | [Football (Soccer) Videos](https://www.scorebat.com/video-api/) | Embed codes for goals and highlights from Premier League, Bundesliga, Serie A and many more | No | Yes | Yes |
 | [Football Prediction](https://boggio-analytics.com/fp-api/) | Predictions for upcoming football matches, odds, results and stats | `X-Mashape-Key` | Yes | Unknown |
 | [Football-Data](https://www.football-data.org) | Football data with matches info, players, teams, and competitions | `X-Mashape-Key` | Yes | Unknown |
+| [Football Standings](https://github.com/azharimm/football-standings-api) | Display football standings e.g epl, la liga, serie a etc. The data is based on espn site | No | Yes | Yes |
 | [JCDecaux Bike](https://developer.jcdecaux.com/) | JCDecaux's self-service bicycles | `apiKey` | Yes | Unknown |
```
Comment (matheusfelipeog) [inline README.md:1053]: Fix the alphabetical order of this API

```suggestion
| [Football Prediction](https://boggio-analytics.com/fp-api/) | Predictions for upcoming football matches, odds, results and stats | `X-Mashape-Key` | Yes | Unknown |
| [Football Standings](https://github.com/azharimm/football-standings-api) | Display football standings e.g epl, la liga, serie a etc. The data is based on espn site | No | Yes | Yes |
| [Football-Data](https://www.football-data.org) | Football data with matches info, players, teams, and competitions | `X-Mashape-Key` | Yes | Unknown |
| [JCDecaux Bike](https://developer.jcdecaux.com/) | JCDecaux's self-service bicycles | `apiKey` | Yes | Unknown |
```

File: README.md [hunk 2]
```diff
@@ -1062,6 +1064,7 @@ API | Description | Auth | HTTPS | CORS |
 | [Tredict](https://www.tredict.com/blog/oauth_docs/) | Get and set activities, health data and more | `OAuth` | Yes | Unknown |
 | [Wger](https://wger.de/en/software/api) | Workout manager data as exercises, muscles or equipment | `apiKey` | Yes | Unknown |
 
+
```
Comment (matheusfelipeog) [inline README.md:1067]: Remove this blank space

```suggestion
```

**Review by matheusfelipeog — CHANGES_REQUESTED** [review]
Thanks for your contribution @azharimm ✋ 

Can you resolve this review, please?

**Review by matheusfelipeog — APPROVED** [review]
Now it's Ok ✔

Thanks for the update ;)

**Pull request merged**